code/obj/machinery/clonepod.dm

```diff
@@ -1143,6 +1143,17 @@
 			message_admins("[key_name(owner)] forced [key_name(target, 1)] ([target == 2 ? "dead" : "alive"]) into \an [grinder] at [log_loc(grinder)].")
 		if (grinder.auto_strip && !grinder.emagged)
 			target.unequip_all()
+			if (length(target.implant))
+				for (var/obj/item/implant/I in target.implant)
+					if (istype(I,/obj/item/implant/projectile))
+						continue
+					var/obj/item/implantcase/newcase = new /obj/item/implantcase(target.loc)
```

Comment:
I think I'd prefer it if it spat the implants out without a case, but that's personal preference and hardly has a gameplay impact

Reply:
The reason I didn't do it without a case is because they're extremely tiny, and easy to miss, meaning the doctor/cloned person might not even realise that their implants popped out. You can always bin the cases later, and its better to have a case left over, than the implant being hard to spot.